tests/Game.class.test.js

```diff
@@ -0,0 +1,965 @@
+/* eslint-disable max-len */
+'use strict';
+
+const Game = require('../src/modules/Game.class');
+
+function sum(game) {
+  let result = 0;
+
+  for (const row of game.getState()) {
+    for (const n of row) {
+      result += n;
+    }
+  }
+
+  return result;
+}
+
+function count(game) {
+  return game.getState().flat().filter(n => n).length;
```

Comment:
The `count` function could be simplified by combining `flat` and `filter` into a single `reduce` call, which could potentially improve performance by only iterating once.